.github/workflows/workflow-pr.yml

```diff
@@ -60,11 +60,32 @@ jobs:
     name: TF Plan Dev Account
     uses: ./.github/workflows/account-deploy.yml
     with:
+      plan_only: true
       workspace_name: development
     secrets:
       aws_access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
       aws_secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+  plan-preproduction-account:
```

Comment:
These environments could be put in a matrix so that they all build and plan simultaneously on PRs. That being said I've never tried that with a workflow call so it might not play ball :man-shrugging: